src/features/channels/components/chat-content/ChatContent.tsx

```diff
@@ -56,28 +49,24 @@ const ChatContent = () => {
 
   return (
     <Paper onScroll={onScroll} ref={containerRef} className={classes.main} square>
-      {sortedMessages.map(
-        (
-          { senderAlias, message, createdAt, senderAddress, replyMessage, replyRecipientAlias },
-          index
-        ) => (
-          <Message
-            // TODO: the backend is not giving us any way to differentiate between messages... Too bad!
-            // eslint-disable-next-line react/no-array-index-key
-            key={index}
-            senderAddress={senderAddress}
-            name={senderAlias}
-            message={message}
-            date={formatDate(createdAt)}
-            color="#A36300"
-          >
-            {/* TODO: there must be a better way to achieve this */}
-            {replyMessage && replyRecipientAlias && (
-              <MessageReply name={replyRecipientAlias} message={replyMessage} color="#A36300" />
-            )}
-          </Message>
-        )
-      )}
+      {sortedMessages.map((message, index) => (
+        <Message
+          // TODO: the backend is not giving us any way to differentiate between messages... Too bad!
+          // eslint-disable-next-line react/no-array-index-key
+          key={index}
+          message={message}
+          color="#A36300"
+        >
+          {/* TODO: there must be a better way to achieve this */}
```

Comment:
If we think that we have some tech debt to resolve, it is a good idea to create tasks and put the id to reference the pending work.